extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -289,7 +294,7 @@ export class ImageViewer {
     // and then naturally upgrade to a higher quality image.
     return timerFor(this.win).promise(1).then(() => {
       this.image_.setAttribute('src', src);
-      return this.loadPromise(this.image_);
```

Comment:
so `BaseElement.loadPromise` seems to be basically a call to `event-helper.loadPromise`. Can't we just import that and use that instead of passing it around? 

I realize `BaseElement.loadPromise` might change, but this call might cause problems the context it's being called with, so if we want to keep this we might need to bind it to the `AmpImageLightbox` before calling it otherwise it might break in the future.